src/Citeproc/Types.hs

```diff
@@ -141,17 +142,23 @@ import Citeproc.Unicode (Lang(..), parseLang, renderLang)
 
 -- | Options affecting the output in ways that go beyond
 -- what can be specified in styles.
-newtype CiteprocOptions =
+data CiteprocOptions =
   CiteprocOptions
   { linkCitations :: Bool
     -- ^ Create hyperlinks from citations to bibliography entries
+  , linkTitles :: Bool
+    -- ^ When a bibliography entry has an identifier (doi, pmcid, pmid, url)
+    -- available, but the style does not explicitly render at least one of them,
+    -- add a hyperlink to the title instead.
   }
   deriving (Show, Eq)
 
 defaultCiteprocOptions :: CiteprocOptions
 defaultCiteprocOptions =
   CiteprocOptions
-  { linkCitations = False }
+  { linkCitations = False 
+  , linkTitles    = True
```

Comment:
Should the default for `linkTitles` be `True` or `False`?

Reply:
Maybe it should be False for now.  If we decide to implement `addHyperlink` for CslJson, then having this as True would affect automated tests.  Pandoc can always set this to True if it wants this by default.